spec/controllers/product_users_controller_spec.rb

```diff
@@ -4,72 +4,85 @@
 RSpec.describe ProductUsersController do
   render_views
 
-  let(:facility) { @authable }
-
-  before(:all) { create_users }
-
-  before(:each) do
-    @authable         = FactoryBot.create(:facility)
-    @facility_account = @authable.facility_accounts.create(FactoryBot.attributes_for(:facility_account))
-    @price_group = FactoryBot.create(:price_group, facility: facility)
-    @instrument       = FactoryBot.create(:instrument,
-                                          facility: @authable,
-                                          facility_account: @facility_account,
-                                          requires_approval: true)
-    @price_policy     = @instrument.instrument_price_policies.create(FactoryBot.attributes_for(:instrument_price_policy).update(price_group_id: @price_group.id))
-    expect(@price_policy).to be_valid
-    @params = { facility_id: @authable.url_name, instrument_id: @instrument.url_name }
-
-    @rule = @instrument.schedule_rules.create(FactoryBot.attributes_for(:schedule_rule))
-    @level = FactoryBot.create(:product_access_group, product_id: @instrument.id)
-    @level2 = FactoryBot.create(:product_access_group, product_id: @instrument.id)
-  end
+  let(:facility) { create(:setup_facility) }
+  let(:instrument) { create(:setup_instrument, facility: facility, requires_approval: true) }
+  let(:guest) { create(:user) }
```

Comment:
I'm not entirely following why this user is named as "guest"--are they not a regular logged in user? Why not just "user"?

Reply:
Yeah, I should clean that up. Before the changes it was `@guest`, which isn't a very good name, but that's one of the default users that gets created by `create_users`. 

I'll do that when I clean up the rubocop stuff.